space-cli/cmd/modules/addons/registry.go

```diff
@@ -37,6 +43,17 @@ func addRegistry(projectID string) error {
 		return nil
 	}
 
+	// check if port is available
+	containerRegistryPort, err = utils.CheckPortAvailability(containerRegistryPort, "registry")
+	if err != nil {
+		return err
+	}
+
+	// sets selected account according to cluster name provided
+	if err := utils.ChangeSelectedAccount(clusterName); err != nil {
+		return utils.LogError("Could not set selected account ", err)
+	}
```

Comment:
We can get the clusterName from the selected account

Reply:
same case as database